### PR TITLE
Nginx ip host

### PR DESCRIPTION
**Pull request by Psalmz777**
**_WHAT DOES THIS PR DO?_**

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: k8s/nginx/development/global-config.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ data:
       set $ip_address $remote_addr;
       
       # Pass domain name and IP address as headers
-      proxy_set_header X-Domain-Name $domain_name;
-      proxy_set_header X-IP-Address $ip_address;
+      proxy_set_header Host $host;
```
Comment (Baalmart) [inline k8s/nginx/development/global-config.yaml:59]: Hi @Psalmz777  , what is the difference here between: `proxy_set_header X-Forwarded-For $remote_addr` and `proxy_set_header X-Real-IP $remote_addr`?

I sense that we are assigning two different headers using the same value? Could I be missing something?

**Comment by Baalmart** [comment]
Hi @Psalmz777 , this has some merge conflicts now. You might want to look into them as we also look forward to capturing the IP and domain name details of each requester

File: k8s/nginx/development/global-config.yaml (hunk 1 above)
Comment (Psalmz777) [reply k8s/nginx/development/global-config.yaml:59]: Lemme slack you on this @Baalmart

**Comment by Psalmz777** [comment]
> Hi @Psalmz777 , this has some merge conflicts now. You might want to look into them as we also look forward to capturing the IP and domain name details of each requester

Alright @Baalmart ,will just raise a new PR

**Pull request closed without merging**